security/pkg/registry/kube/serviceaccount_test.go

```diff
@@ -35,6 +36,17 @@ func createServiceAccount(name, namespace string) *v1.ServiceAccount {
 	}
 }
 
+func createMissedServiceAccount(name, namespace string) *cache.DeletedFinalStateUnknown {
+	return &cache.DeletedFinalStateUnknown{
```

Comment:
searched thru the PR, this `DeletedFinalStateUnknown` only appears in the test code?

Reply:
please see the documentation for possible returned values:
https://github.com/kubernetes/client-go/blob/master/tools/cache/controller.go#L166-L184
> OnDelete will get the final state of the item if it is known, otherwise it will get an object of type DeletedFinalStateUnknown.